.circleci/config.yml

```diff
@@ -39,13 +104,21 @@ jobs:
             fi
             ./autogen.des --quiet
             make dist -j 16
+            mkdir -p ~/workspace/tools
```

Comment:
why the switch to a subdir? since we know exactly what to save, why not grab them in place?

Reply:
I've tried to keep the old file structure.
`varnish-x.y.z.tar.gz` and `.is_weekly` are stored in `~/workspace/`. 
But `mv tools/*.suppr ~/workspace/tools/` copies just the `.suppr` files, not all files under `tools/`.

Reply:
true, but you are already using a glob pattern in `persist_to_workspace` to only capture the `suppr` files
it's true that you need it for `store_artifacts` since you don't know the name of the files buI would argue that we possibly don't need the files as artifacts though, and we absolutely don't care about the `suppr` files or `.weekly` as artifacts.

Reply:
The use the artifacts in the intermediate steps as a debug tool - this is the only way I've found to be able to inspect what a given step produces as an outcome. I can remove it if it is confusing.
Only `collect_packages` step produces the final packages.tar.gz with all packages inside it as a meaningful artifact.

Reply:
since we can rerun a job and log into the container, I don't feel the artifacts are that useful. A borked file is often not enough and you need to see the broader environment, so I'd do away with intermediate file

Reply:
I'll remove the usage of store_artifacts in the intermediate steps!